packages/ai-jsx/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "./tsconfig-base.json",
   "include": ["src/**/*", ".eslintrc.cjs"],
   "compilerOptions": {
-    "outDir": "dist/esm"
+    "outDir": "dist/esm",
+    "rootDir": "./src"
```

Comment:
Just to be extra safe, after we land this PR, let's publish ai-jsx and make sure it's still usable when installed from a third-party library.